src/Surfnet/StepupMiddleware/ApiBundle/Controller/RaCandidateController.php

```diff
@@ -38,36 +39,44 @@ class RaCandidateController extends Controller
     private $raCandidateService;
 
     /**
-     * @var InstitutionRoleSet
+     * @var InstitutionAuthorizationService
      */
-    private $roleRequirements;
+    private $authorizationService;
 
-    public function __construct(RaCandidateService $raCandidateService)
-    {
+    public function __construct(
+        RaCandidateService $raCandidateService,
+        InstitutionAuthorizationService $authorizationService
+    ) {
         $this->raCandidateService = $raCandidateService;
 
         $this->roleRequirements = new InstitutionRoleSet(
-            [new InstitutionRole(InstitutionRole::ROLE_SELECT_RAA)]
+            [new InstitutionRole(InstitutionRole::ROLE_USE_RAA)]
```

Comment:
You removed the `roleRequirements` field, but stil set it with an updated role. I'm confused here. Imho the field instantiation can be removed as it is never used in the code.

Reply:
Good catch, removed the line